nft_ingester/src/api/api_impl.rs

```diff
@@ -90,6 +93,7 @@ where
         storage_service_base_path: Option<String>,
         token_price_fetcher: Arc<TPF>,
         native_mint_pubkey: String,
+        max_query_statement_timeout_sec: u64,
```

Comment:
inconsistent naming: everywhere else the naming is "api_query_max_statement_timeout_secs"